packages/blockchain-wallet-v4-frontend/src/providers/AnalyticsTracker/index.js

```diff
@@ -44,6 +49,7 @@ class AnalyticsTracker extends React.PureComponent {
       <Iframe
         id='matomo-iframe'
         src={walletHelper + '/wallet-helper/matomo/#/'}
+        innerRef={this.iframe}
```

Comment:
`innerRef` is deprecated in styled-components v4 btw
https://www.styled-components.com/docs/api#innerref-prop